whrbcatalog/application.py

```diff
@@ -1,15 +1,16 @@
-# Final Project: Chris Sun, Joyce Lu, Simon Eder
```

Comment:
This looks like a lot of changes, but I didn't actually change any functionality. I just removed some unused code and fixed flake8 and pylint errors.